_includes/setup/get-sdk-win.md

```diff
@@ -0,0 +1,59 @@
+## Get the Flutter SDK
+
+To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path.
+Running `flutter doctor` shows any remaining dependencies you may need to install.
+
+### Clone the repo
+
+If this is the first time you're installing Flutter on this machine, clone the
+repository and then add the `flutter` tool to your path:
```

Comment:
Can we mention that you should do this in Cmd or PowerShell? We currently don't support third-party shells (like Git Bash or Bash on Ubuntu on Windows).